docs/Ecosystem/projects.md

```diff
@@ -6,23 +6,70 @@
+- **[Peal](https://github.com/Pearl-Market/marketplace)**: Discover and share typefaces and designs on-chain.
+- **[DFSP](https://github.com/shazi57/dfsp-js)**: Decentralized, private live-streaming platform for creators.
+- **[Decensus](https://github.com/decensus-crypto/app)**: Web3-based survey platform.
+- **[creativegene](https://github.com/ddresch/creativegene)**: Decentralized social platform for buying and selling digital goods.
+- **[Muzu](https://ethglobal.com/showcase/muzu-3jhzc)**: Content-locked music NFTs.
 
 ### Infrastructure
 
-- **[Cask](https://www.cask.fi/)**: Automated, recurring payments.
-- **[Krebit](https://docs.krebit.id/#/?id=main)**: Open identity verification protocol and marketplace for Verifiable Credentials.
+- **[BAILOUT](https://ethglobal.com/showcase/bailout-ia6s1?ref=spark.litprotocol.com)**: Securely move assets between EOAs, smart contract wallets, and cold wallets.
+- **[Strata Protocol](https://strataprotocol.com/)**: An open-source protocol to launch tokens around a person, project, idea, or collective on Solana.
+- **[Key-Recovery](https://github.com/Joseph-Gross/key-recovery)**: Distributed social private key recovery made by the team at [Sling Protocol](https://twitter.com/slingprotocol).
+- **[Neya](https://github.com/stefsimion/Site-Neya-2)**: Web3-native 2FA protocol.
+- **[EtherMonitor](https://github.com/seetadev/EtherMonitor)**: Tools for traffic incident management.
+- **[ourpass](https://ethglobal.com/showcase/ourpass-91ucq)**: NFT event pass generator.
+
+### Data and Identity Marketplaces
+
+- **[WeaveDB](https://docs.weavedb.dev/docs/examples/lit-protocol)**: Decentralized Web3 database.
+- **[Streamr](https://blog.streamr.network/streamr-integrates-lit-protocol/)**: Distributed real-time data network.
+- **[KaratDAO](https://docs.karatdao.com/why-karat-network)**: A decentralized data network connecting Web2 to Web3.
+- **[Cheqd](https://cheqd.io/)**: Infrastructure for trusted data markets.
+- **[Krebit](https://spark.litprotocol.com/krebitxlitactions/)**: Open identity verification protocol and marketplace for Verifiable Credentials.
+- **[AQA Protocol](https://ethglobal.com/showcase/aqa-protocol-b5ok4?ref=spark.litprotocol.com)**: Give users full control over their private data.
+- **[VerifyXYZ](https://github.com/verify-xyz/did-profiles/tree/poc-lit?ref=spark.litprotocol.com)**: Essential building blocks for Decentralized ID.
+
+
+### Privacy and Encryption
+
+- **[dKeeper](https://ethglobal.com/showcase/dkeeper-3ma7r?ref=spark.litprotocol.com)**: Access control management for dApps.
+- **[Selfguard](https://github.com/selfguard/selfguard?ref=spark.litprotocol.com)**: Universal API for encryption.
+- **[Starling Lab](https://github.com/starlinglab/archive-explorer/)**: Token-gated data archives.
+- **[Curve Labs](https://github.com/Curve-Labs/lit-privacy/tree/main)**: Privacy-preserving infrastructure tooling.
```

Comment:
A better description for the curve labs project is, "Tooling for private on-chain voting"